### ISSUE TITLE
FIXME: Enable some systemd services later because of grub-install ERROR:

### ISSUE THREAD

**Issue by LeoMeinel** [LeoMeinel/mdadm-encrypted-btrfs#11]
```
Detecting snapshots ...
mount: /tmp/grub-btrfs.<...>: special device /dev/disk/by-uuid/<UUID of /dev/mapper/md0_crypt> does not exist.
```

**Comment by LeoMeinel** [comment]
Added wontfix because this probably isn't an issue with the script and there is really no way to not run into this during installation.
I don't even know if the services or grub are the problem, so I can just hope that over time this gets fixed in one of them.

**Comment by LeoMeinel** [comment]
This pretty much doesn't matter. Even tho it is probably a bug, I'll just close the issue.
This only occurs during install and it can be easily overcome!

**Comment by leomeinel** [comment]
This was fixed by #142
